external/tidal/client_test.go

```
@@ -91,6 +91,44 @@ func TestFetchListMaxItems(t *testing.T) {
 	}
 }
 
+func TestUserPlaylistsRespectsPageCap(t *testing.T) {
+	// The playlistsAndFavoritePlaylists endpoint rejects limit > 50, unlike
+	// most v1 list endpoints which allow 100.
+	total := 120
+	srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		limit, _ := strconv.Atoi(r.URL.Query().Get("limit"))
+		if limit > 50 {
+			w.WriteHeader(http.StatusBadRequest)
+			fmt.Fprint(w, `{"status":400,"subStatus":1001,"userMessage":"Too big page, max page size is [50]"}`)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprint` is not checked

(errcheck)